__tests__/components/editor/property/PropertyLabelInfo.test.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import 'jsdom-global/register'
+import React from 'react'
+import { shallow } from 'enzyme'
+import PropertyLabelInfo from 'components/editor/property/PropertyLabelInfo'
+import { OverlayTrigger } from 'react-bootstrap'
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
```

Comment:
How about the external link icon (https://fontawesome.com/icons/external-link-alt?style=solid) rather than the info circle?

Reply:
It's not always external; sometimes it is a popover.

Reply:
What I'm suggesting is a external link icon for the external links and an info circle for the popover.  Different icons should indicate different behavior.

Reply:
Done @jcoyne